### DISCUSSION TITLE
CWMS_GAGE to CDA roadmap

### DISCUSSION THREAD

**Discussion by JeremyDKellett** [USACE/cwms-data-api#814] [General]
Placeholder for convo on roadmap for CWMS_GAGE implementation into CDA. I have a application to modernize that could use the sensor structure in CWMS_GAGE.STORE_GAGE and .STORE_GAGE_SENSOR and am curious of a timeline to CDA implementation.

Does anybody use CWMS_GAGE to store their gage/sensor configs?

**Comment by MikeNeilson** [comment]
We currently have no definitive timeline on implementing endpoints for this. However, we if it's time critical we will gladly help onboard anyone into the development process. Or except contributions from any vendor willing to follow the defined rules.

HEC won't be able to do any contract awards until next year, so if you want it done sooner, I'd strongly suggest in-house if you have the time.

That all said, anyone who reads this please answer Jeremy's question. If multiple districts *desire* to use this, we will take the time to help get it done.

@krowvin Can you run this by the usual crowd's to see who wants better support to gauge (sorry, I hate that misspelling) inventory, tracking, and maintenance?
